test/spec/app_spec.rb

```diff
@@ -830,6 +788,150 @@
+      page.edit_user_role(subdomain: @team1_slug)
+      sleep 5
+      expect(@driver.page_source.include?('Editor')).to be(true)
+    end
+
+    it "should delete member from team", bin4:true do
+      page = Page.new(config: @config, driver: @driver)
+      page.go(@config['api_path'] + '/test/session?email='+@user_mail)
+      #As the group creator, go to the members page and approve the joining request.
+      sleep 1
+      page = TeamsPage.new(config: @config, driver: @driver).load
+          .delete_user_team(subdomain: @team1_slug)
+
+      expect(@driver.find_elements(:xpath => '//button').length == 4).to be(true)
+    end
+
+    it "should delete tag from tags list (for media and source)", bin3:true  do
+    #   source implemented at "should add and remove source tags"
+      page = api_create_team_project_and_claim_and_redirect_to_media_page
+      @driver.find_element(:css, '.media-detail').click
+      @driver.find_element(:css, '.media-actions').click
+      #@driver.find_element(:class, 'media-actions__menu-item').click
+      @driver.find_element(:css,'.media-actions__edit').click
+      sleep 3
+      fill_field('.ReactTags__tagInput input', "TAG")
+      @driver.action.send_keys("\n").perform      
+      sleep 1
+      fill_field('.ReactTags__tagInput input', "TAG2")
+      @driver.action.send_keys("\n").perform
+      expect(page.tags.length == 2).to be(true)
+      list = @driver.find_elements(:class, "ReactTags__remove")
+      list[1].click
+      sleep 3
+      expect(page.tags.length == 1).to be(true)
+    end
+
+    it "should show 'manage team' link only to team owners", bin3: true do
+      user = api_register_and_login_with_email
+      team = request_api 'team', { name: "team#{Time.now.to_i}", email: user.email, slug: "team#{Time.now.to_i}" }
+      user2 = api_register_and_login_with_email
+      page = TeamsPage.new(config: @config, driver: @driver).load
+      page.ask_join_team(subdomain: team.slug)
+      @wait.until {
+        expect(@driver.find_element(:class, "message").nil?).to be(false)
+      }
+      api_logout
+      @driver = new_driver(webdriver_url,browser_capabilities)
+      page = Page.new(config: @config, driver: @driver)
+      page.go(@config['api_path'] + '/test/session?email='+user.email)
+      #As the group creator, go to the members page and approve the joining request.
+      page = TeamsPage.new(config: @config, driver: @driver).load
+          .approve_join_team(subdomain: team.slug)
+      api_logout
+
+      @driver = new_driver(webdriver_url,browser_capabilities)
+      page = Page.new(config: @config, driver: @driver)
+      page.go(@config['api_path'] + '/test/session?email='+user2.email)
+      page = TeamsPage.new(config: @config, driver: @driver).load
+      sleep 3
+      @driver.find_element(:class,"header-actions__menu-toggle").click
+      sleep 3
+      expect(@driver.page_source.include?('Manage team')).to be(false)
```

Comment:
Here you test that "Manage team" is not displayed to a team owner but doesn't test if it is displayed to a team owner.

Reply:
Please also test if a team owner can actually see the "Manage team" option.